.github/workflows/main.yml

```diff
@@ -7,6 +7,7 @@ on:
     branches:
       - main
   pull_request:
+  workflow_dispatch:
```

Comment:
Why this change?

Reply:
It allows manually triggering the workflow from UI, notably the workflow can be run from a branch on a fork to test changes to the workflow before sending a PR. I can revert this if needed, but I think it makes development of this file safer without any real downside

Reply:
great ... seems like that should be the default.